code/game/machinery/firealarm.dm

```diff
@@ -460,6 +460,9 @@
 		. += "The local area thermal detection light is [my_area.fire_detect ? "lit" : "unlit"]."
 		. += "<b>Left-Click</b> to activate all firelocks in this area."
 
+	if (issilicon(user))
+		. += span_notice("Ctrl-Click the fire alarm to toggle thermal sensors, which controls auto-activation.")
```

Comment:
This should be a contextual screentip. We should add those to fire alarms if we don't already